### ISSUE TITLE
No values in the energy tab homey 3.0.0

### ISSUE THREAD

**Issue by driesk81** [biemond/com.athom.zevercloud#2]
Woud it be possible to show the generated power in the energy tab in homey 3.0.0?

**Comment by biemond** [comment]
let me check it out, I heard about this but did not investigate just yet

**Comment by biemond** [comment]
So I got it already working, I will test it tomorrow when there is some sun

**Comment by biemond** [comment]
submitted for review , now waiting for the approval

**Comment by driesk81** [comment]
You are fast :-)
I will try to install via cli tomorrow.

Thanks for your effort.

**Comment by biemond** [comment]
yeah, was easy, update the app.json and had to rename 1 sensor . that is all

![Screenshot_2019-11-02-12-29-14](https://user-images.githubusercontent.com/1085004/68076667-fef7b580-fdb7-11e9-9ba2-dd322f0a1828.png)

**Comment by driesk81** [comment]
I just installed via cli method, works fine now. Nice Job!
